src/Text/Pandoc/Options.hs

```diff
@@ -254,7 +271,8 @@ data WriterOptions = WriterOptions
   , writerListings          :: Bool       -- ^ Use listings package for code
   , writerHighlightStyle    :: Maybe Style  -- ^ Style to use for highlighting
                                            -- (Nothing = no highlighting)
-  , writerSetextHeaders     :: Bool       -- ^ Use setext headers for levels 1-2 in markdown
+  -- | Header format to use in markdown
```

Comment:
Instead of this line, use `-- ^` on the line with the option, as in the other cases.